NetCord/Rest/MessagePollAnswerCount.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+
+using NetCord.JsonModels;
+
+namespace NetCord.Rest;
+
+public partial class MessagePollAnswerCount : JsonEntity
+{
+    [JsonPropertyName("id")]
+    public int AnswerId { get; set; }
```

Comment:
It's a duplicate of the `Id` property defined in `JsonEntity` and I think it should also be an `ulong`.

Reply:
Yup already changed that in my local commit